src/code/code_800EC960.c

```diff
@@ -2820,31 +2837,35 @@ void func_800F3A08(u8 bankIdx, u8 entryIdx, u8 channelIdx) {
         phi_a1 = -1;
     }
 
+    // CHAN_UPD_SCRIPT_IO (slot 2)
     Audio_QueueCmdS8(0x6020000 | (channelIdx << 8) | 2, phi_a1);
-    if (reverb != D_8016B8B8[channelIdx].unk_08) {
+    if (reverb != D_8016B8B8[channelIdx].oldReverb) {
         Audio_QueueCmdS8(0x5020000 | (channelIdx << 8), reverb);
-        D_8016B8B8[channelIdx].unk_08 = reverb;
+        D_8016B8B8[channelIdx].oldReverb = reverb;
     }
-    if (freqScale != D_8016B8B8[channelIdx].unk_04) {
+    if (freqScale != D_8016B8B8[channelIdx].oldFreqScale) {
         Audio_QueueCmdF32(0x4020000 | (channelIdx << 8), freqScale);
-        D_8016B8B8[channelIdx].unk_04 = freqScale;
+        D_8016B8B8[channelIdx].oldFreqScale = freqScale;
```

Comment:
I think `prev` (or `cur` depending on how you look at it) is better than `old`

Reply:
dropped the prefix completely